src/styles/main.scss

```diff
@@ -0,0 +1,67 @@
+$padding: 10px;
+$gap: 1px;
+$day-color: #eee;
+$day-color-hover: #ffbfcb;
+$day-height: 100px;
+$day-width: 100px;
+$calendar-width: $day-width * 7 + $gap * 6;
+$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+
+html {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+.calendar {
```

Comment:
According to BEM rules, each BEM block should be in a separate file
Also, you can move variables to a separate file as well